playbooks/provision.yaml

```diff
@@ -7,18 +7,18 @@
   vars:
     openshift_master_url: "{{ lookup('env','OCP_URL') }}"
     token: "{{ lookup('env','OCP_TOKEN') }}"
-    THOTH_BACKEND_NAMESPACE: "thoth-test-back"
-    THOTH_MIDDLEEND_NAMESPACE: "thoth-test-middle"
-    THOTH_FRONTEND_NAMESPACE: "thoth-test-front"
+    THOTH_BACKEND_NAMESPACE: "goern-dev-thoth"
+    THOTH_MIDDLEEND_NAMESPACE: "goern-dev-thoth"
+    THOTH_FRONTEND_NAMESPACE: "goern-dev-thoth"
+    THOTH_DEPLOYMENT_NAME: "goern-dev-thoth"
     THOTH_CEPH_SECRET_KEY: "{{ lookup('env','THOTH_CEPH_SECRET_KEY') }}"
     THOTH_CEPH_KEY_ID: "{{ lookup('env','THOTH_CEPH_KEY_ID') }}"
-    THOTH_CEPH_HOST: "{{ lookup('env','THOTH_CEPH_HOST') }}"
+    THOTH_CEPH_HOST: "{{ lookup('env','THOTH_S3_ENDPOINT_URL') }}"
```

Comment:
Why there was a need to rename this?

Reply:
Because the setting is not a Host but a URL and it is not a Ceph host but a S3 Endpoing :)

Reply:
I would rather have these unified not to introduce hard to tackle aliases. We can fix that in the code.

Reply:
Right... we should comply to S3 nameing etc...